portfolio/src/main/webapp/index.html

```diff
@@ -190,6 +191,12 @@ <h3>Teaching</h3>
               </li>
           </ul>
       </article>
+      <article id="randomFacts">
+        <h2>Fun facts</h2>
+        <p>Push the bottom to get a random fun fact about me</p>
+        <button id="btn-fact" type="button" onClick="addRandomFact()">Click Me!</button>
```

Comment:
Consider combining these to make a self contained button that describes it's purpose, rather than a paragraph and a button.

Some screenreader users might not even hear the paragraph because they'll navigate by controls, so they'll skip straight to the button and hear "Click Me! double tap to activate" and it will not be descriptive enough.

Reply:
+1 on this!

Reply:
Excellent recommendation, I was not aware of that.